frontend/src/App.svelte

```diff
@@ -200,26 +205,28 @@
 
 <div class="m-0 h-screen bg-base-300">
   <div
-    class="flex h-fit right-0 bottom-0 fixed align-bottom justify-center items-end"
+    class="flex left-0 bottom-0 fixed align-bottom justify-center items-end"
     style="z-index:1"
   >
-    <div class="block m-6">
-      <ul class="menu w-16 py-3 shadow-lg bg-base-100 rounded-box">
+    <div class="grid block h-screen m-6 place-items-end">
+      <ul class=" menu w-16 py-3 shadow-lg bg-base-100 rounded-box">
         <li>
-          <a class="" class:btn-active={show_menu} on:click={toggleMenu}>
+          <a
+            class=" focus:bg-primary"
+            class:btn-active={show_filter}
+            on:click={toggleFilter}
+          >
             <svg
```

Comment:
It would be nicer to have all SVG files in a asset folder on the front end and call them here form there. It would make it easier to maintain and read I think :)

Reply:
(I only highlighted one sag here, but this suggestion applies to all of them)

Reply:
The svgs are actually hard coded. The code: `xmlns="http://www.w3.org/2000/svg"` is only the namespace for the svg file, which state specification and syntax the svg file follows. The documentation regarding the namespace could be found at [https://www.w3.org/TR/SVG/](url). It just like the xml file declare the template it following at the begin.

Reply:
I mean, you can take each <svg></svg> blocks representing an icon, move them into a `filter_icon.svg`file and call that svg in the App.svelte using an image tag referring to that file.

However, that could lead to performance issues since it creates dubble request and loading so it could be nice to cache it. 
An alternative is to make each button a component and import it, which might be the better way to go here if that make sense. I just feel this file will get very difficult to maintain and read while we continue adding buttons.

Reply:
I see your points.  There are no differences in performance between inline svg and seperate svg file in our case. [https://stackoverflow.com/a/27666979](url).  We can have a sidebar component for sure, still to manage several svgs either with seperate files or inline will causing problems. I can add alter values to those svgs so that make it easier to identify them in the code.